### ISSUE TITLE
Using @directus/sdk in React Native gives URL.pathname not implemented error

### ISSUE THREAD

**Issue by Popl7** [directus/directus#19288]
### Describe the Bug

I am trying to use @directus/sdk with graphql and authentication in a React Native app.
Logging in fails with this error: `URL.pathname not implemented error`.

What can we do to use Directus in a React Native app?

### To Reproduce

Start a new React Native app:
`npx react-native@latest init mytestapp`

Add NPM package:
`yarn add @directus/sdk`

Add directus to the app:
```
const directus = createDirectus<Schema>('http://localhost:8055')
  .with(graphql())
  .with(authentication('json'));
```

In the App.tsx add:
```
const res = await directus
      .login(email, password)
      .then(() => {
        console.debug('[Auth] logged in');
      })
      .catch((error: any) => {
        console.error('[Auth] logging error...', error);
        return error;
      });
```

### Directus Version

v11.0.0

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by br41nslug** [comment]
Native apps are always tricky 🤔 In this case it looks like react-native is missing a polyfill/shim (or however they call it) for the URL API.

With a quick google you can find a couple of these https://github.com/charpeni/react-native-url-polyfill

**Comment by Popl7** [comment]
ah, yes. That worked 🥳 
Thank you very much!

**Comment by kellvembarbosa** [comment]
@Popl7 can you share how it worked?

**Comment by Popl7** [comment]
yes, of course. Sorry for not writing it up.

I installated [react-native-url-polyfill](https://github.com/charpeni/react-native-url-polyfill):
- `yarn add react-native-url-polyfill` or `npm install react-native-url-polyfill`

Then I added `import 'react-native-url-polyfill/auto';` at the top of my index.js file.

This was enough to make the polyfill work before the other code was loaded.
The Directus SDK can be used without the Url error.

Let me know if you have questions about this.
